static/index-web.html

```diff
@@ -2,7 +2,66 @@
 <html>
   <head>
     <meta charset="utf-8" />
-    <script src="/ui.js" async></script>
+    <script src="/public/ui.js" async></script>
+    <link rel="icon" type="image/png" href="/public/favicon.png" />
```

Comment:
(presumably if we want to experiment, this can be pushed and we can check in analytics for page requests, so if you deploy this please set a note or reminder to check analytics a few days after)

Reply:
It looks like it's just needed for IE. I can add an additional icon for that.